package chap17.ex;

import java.io.IOException;

public class Main17_4 {

	public static void main(String[] args) throws Exception {
		// TODO 自動生成されたメソッド・スタブ

		throw new IOException();
	}

}
